apps/meteor/app/version-check/server/index.ts

```diff
@@ -12,15 +12,15 @@ if (SyncedCron.nextScheduledAtDate(jobName)) {
 	SyncedCron.remove(jobName);
 }
 
-const addVersionCheckJob = Meteor.bindEnvironment(() => {
+const addVersionCheckJob = () => {
```

Comment:
I was expecting synced cron to require bindEnvironment 😳

Reply:
That's correct, I'll add it back.